chatbot/app.py

```
@@ -1,94 +1,129 @@
+MODEL_PATH = os.path.join(BASE_DIR, "data.pth")
+
+################################
+# NLTK SAFE SETUP (3.8.2+)
+################################
+def ensure_nltk_tokenizers():
+    try:
+        nltk.data.find("tokenizers/punkt_tab/english")
+    except LookupError:
+        try:
+            nltk.download("punkt_tab")
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)